client/blocks/stats-sparkline/index.jsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { isJetpackSite, getSiteDomain } from 'state/sites/selectors';
+
+const StatsSparkline = ( { isJetpack, siteDomain, className, siteId } ) => {
+	if ( ! siteId || ! siteDomain || isJetpack ) {
+		return null;
+	}
+
+	return (
+		<img
+			className={ className }
+			src={ `https://${ siteDomain }/wp-includes/charts/admin-bar-hours-scale-2x.php?masterbar=1&s=${ siteId }` } />
+	);
+};
+
+StatsSparkline.propTypes = {
+	className: PropTypes.string,
+};
+
+export default connect(
+	( state, ownProps ) => {
+		const { siteId } = ownProps;
```

Comment:
> An alternative would be to rely on the selectedSiteId

I had originally gone that route but thought at somepoint the sparkline could be used on pages like `/stats` too where multiple sites are listed.

If `siteId` is `null` the component returns `null`.  I could see `QuerySite` being worthwhile potentially but for the current use, and potentially adding to the stats listing page - it might not be needed in either case.